.buildkite/test-e2e.yml

```
@@ -10,11 +10,13 @@
     - bash ../.buildkite/build-start-operator.sh
     - kubectl wait --timeout=90s --for=condition=Available=true deployment kuberay-operator
     # Run e2e tests and print KubeRay operator logs if tests fail
-    - echo "--- START:Running e2e rayservice (nightly operator) tests"
+    - echo "--- START:Running e2e (nightly operator) tests"
     - if [ -n "${KUBERAY_TEST_RAY_IMAGE}"]; then echo "Using Ray Image ${KUBERAY_TEST_RAY_IMAGE}"; fi
     - set -o pipefail
-    - KUBERAY_TEST_TIMEOUT_SHORT=1m KUBERAY_TEST_TIMEOUT_MEDIUM=5m KUBERAY_TEST_TIMEOUT_LONG=10m go test -timeout 30m -v ./test/e2e 2>&1 | awk -f ../.buildkite/format.awk || (kubectl logs --tail -1 -l app.kubernetes.io/name=kuberay && exit 1)
-    - echo "--- END:e2e rayservice (nightly operator) tests finished"
+    - mkdir -p "$(pwd)/tmp" && export KUBERAY_TEST_OUTPUT_DIR=$(pwd)/tmp
+    - echo "KUBERAY_TEST_OUTPUT_DIR=$$KUBERAY_TEST_OUTPUT_DIR"
+    - KUBERAY_TEST_OUTPUT_DIR=$$KUBERAY_TEST_OUTPUT_DIR KUBERAY_TEST_TIMEOUT_SHORT=1m KUBERAY_TEST_TIMEOUT_MEDIUM=5m KUBERAY_TEST_TIMEOUT_LONG=10m go test -timeout 30m -v ./test/e2e 2>&1 | awk -f ../.buildkite/format.awk | tee $$KUBERAY_TEST_OUTPUT_DIR/gotest.log || (kubectl logs --tail -1 -l app.kubernetes.io/name=kuberay | tee $$KUBERAY_TEST_OUTPUT_DIR/kuberay-operator.log && find "$$KUBERAY_TEST_OUTPUT_DIR" -name "*.log" | tar --transform='s|^.*/tmp/||' -cf /artifact-mount/e2e-log.tar -T - && exit 1)
```

```suggestion
    - KUBERAY_TEST_TIMEOUT_SHORT=1m KUBERAY_TEST_TIMEOUT_MEDIUM=5m KUBERAY_TEST_TIMEOUT_LONG=10m go test -timeout 30m -v ./test/e2e 2>&1 | awk -f ../.buildkite/format.awk | tee $$KUBERAY_TEST_OUTPUT_DIR/gotest.log || (kubectl logs --tail -1 -l app.kubernetes.io/name=kuberay | tee $$KUBERAY_TEST_OUTPUT_DIR/kuberay-operator.log && find "$$KUBERAY_TEST_OUTPUT_DIR" -name "*.log" | tar --transform='s|^.*/tmp/||' -cf /artifact-mount/e2e-log.tar -T - && exit 1)
```

You've already exported this variable above. Either remove the `export` above or remove the variable here. Ditto for other commands.

---

I removed the export above, thanks. https://github.com/ray-project/kuberay/pull/3405/commits/38f90107214cb2143ffd7e75c4fafc8fe5cc13e3

---

Fixed in https://github.com/ray-project/kuberay/pull/3405/commits/f8694e989c8c7b1f3d5ed1e25edd7a059c0f1f82, thanks!